File: python/Ram.py
class Ram():

    # Constructor
    def __init__(self):
        self.capacity = 0
        self.price = 0
    
    # Setter and Getter
    def setCapacity(self, capacity):
        self.capacity = capacity
    
    def getCapacity(self):
        return self.capacity
    
    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price
    
    # Menampilkan atribut kelas
    def showRam(self):
        print("Ram Capacity    : " + str(self.capacity) + " Gb" )
        print("Ram Price       : Rp. " + str(self.price))

File: java/Ram.java
public class Ram {
    
    private int capacity;
    private int price;

    // Constructor
    Ram(){
    }
    Ram(int c, int p){
        capacity = c;
        price = p;
    }

    // Getter and Setter
    public int getCapacity() {
        return capacity;
    }
    public int getPrice() {
        return price;
    }
    public void setPrice(int price) {
        this.price = price;
    }
    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    //Menampilkan atribut kelas Ram
    public void showRam() {
        System.out.println("Ram Capacity    : " + capacity + " Gb" );
        System.out.println("Ram Price       : Rp. " + price);
    }
    
}
